src/app/[locale]/specs/oauth/en.mdx

```diff
@@ -256,7 +256,10 @@ Additional requirements:
 - confidential clients should periodically rotate client keys, adding new keys to the JWKS set and using then for new sessions, then removing old keys once they are no longer associated with any active auth sessions
 - confidential clients must include `token_endpoint_auth_method` as `private_key_jwt` in their client metadata document
 - confidential clients are expected to immediately remove client authentication keys from their client metadata if the key has been leaked or compromised
-- Authorization Servers must bind active auth sessions for confidential clients to the client authentication key used at the start of the session. The server should revoke the session and reject further token refreshes if the client authentication key becomes absent from the client metadata. This means the Authorization Server is expected to periodically re-fetch client metadata.
+- Authorization Servers must bind active auth sessions for confidential clients to the client authentication key used at the start of the session. They do so by ensuring that the key's identifier (`kid`), signature algorithm (`alg`) and thumbprint of the signing key (`jkt`) of the client attestation JWT stay the same for the entire life time of the session. The server must revoke the session and reject further token refreshes if the client authentication key becomes absent from the client metadata. This means the Authorization Server must periodically re-fetch client metadata (and related keyset).
+- The client assertion JWT must include an `iat` claim (issued at) timestamp. Authorization Servers should not reject client assertion JWTs generated less than a minute ago.
+- The client assertion JWT must include an `jti` claim (JWT ID) with a unique value to prevent replays. Authorization Servers must ensure uniqueness of `jti` values.
```

Comment:
should we mention that this uniqueness is only relevant during the validity period of the token? eg, between "issued at" and "expires at" (plus any clock fuzzy window)